### PR TITLE
Adding wxmplot & wxutils reciple

### PR DESCRIPTION
**Pull request by kbeeperez**
implements:
https://github.com/nsls-ii-forge/wxutils-feedstock
https://github.com/nsls-ii-forge/wxmplot-feedstock

Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by kbeeperez** [comment]
Attn @mrakitin

File: recipes/wxmplot/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+  number: 0
+  script: {{ PYTHON }} -m pip install . --no-deps -vv
+
+requirements:
+  build:
+    - python >=3.6
+    - pip
+  run:
+    - python >=3.6
+    - matplotlib >=2.0
```
Comment (mrakitin) [inline recipes/wxmplot/meta.yaml:23]:
```suggestion
    - matplotlib-base >=2.0
```

**Review by mrakitin — CHANGES_REQUESTED** [review]
@kbeeperez, thanks! For the Linux tests to pass, you'll need to add https://github.com/nsls-ii-forge/wxmplot-feedstock/blob/master/recipe/yum_requirements.txt (as a raw file). Also, `matplotlib` -> `matplotlib-base` seems to be reasonable here as it uses a different backend than pyqt.

To @conda-forge/staged-recipes, I agree to be listed as a maintainer.

File: recipes/wxmplot/yum_requirements.txt [hunk 2]
```diff
@@ -0,0 +1,2 @@
+gtk2-devel
+libXxf86vm
```
Comment (mrakitin) [inline recipes/wxmplot/yum_requirements.txt:2]: This file may be needed in the `wxutils` recipe dir too, as eventually, they will appear in separate feedstock repositories.

Comment (mrakitin) [reply recipes/wxmplot/yum_requirements.txt:2]: We have it in both feedstocks:
- https://github.com/nsls-ii-forge/wxmplot-feedstock/blob/master/recipe/yum_requirements.txt
- https://github.com/nsls-ii-forge/wxutils-feedstock/blob/master/recipe/yum_requirements.txt

Comment (kbeeperez) [reply recipes/wxmplot/yum_requirements.txt:2]: Alright I'll add it.

**Review by mrakitin — APPROVED** [review]

**Review by dopplershift — APPROVED** [review]

**Comment by mrakitin** [comment]
Thank you, @dopplershift!

**Pull request merged**